### DISCUSSION TITLE
How can I make the alignment vertically perfect?

### DISCUSSION THREAD

**Discussion by ArchieCoder** [unoplatform/uno#17485] [General]
I'm not able to have good alignment for all platforms. Here is the result with the default LineStackingStrategy and with BlockLineHeight. I'm using the Google Font Inter (variable font).

<img width="1386" alt="image" src="https://github.com/user-attachments/assets/83255fe4-43cf-4325-a248-c8dc1fb0706c">

Associated XAML:

```
<StackPanel
    BorderBrush="{StaticResource SidePanelBorderBrush}"
    BorderThickness="1"
    CornerRadius="6"
    Height="32"
    Margin="12,0"
    Orientation="Horizontal"
    Padding="8"
    Spacing="4">

    <Ellipse
        Fill="Green"
        Height="8"
        Width="8" />

    <TextBlock
        FontFamily="{StaticResource MainFont}"
        FontSize="12"
        FontWeight="500"
        Foreground="{StaticResource Gray700Brush}"
        LineHeight="18"
        LineStackingStrategy="BlockLineHeight"
        Text="Commissioned"
        VerticalAlignment="Center" />

</StackPanel>
```

**Comment by jeromelaban** [comment]
Thanks for the question. Is the issue specifically with how WinUI handles the underlined section? If so, it may be a different in the rendering of the font itself on WinUI. Have you tried with Open Sans to see if it behaves the same?

**Comment by ArchieCoder** [comment]
Good news: using OpenSans without the LineStackingStrategy the render is great.

Note: on WinUI, the font looks very bold versus Skia targets.

WinUI
<img width="100" alt="OpenSans - WinUI" src="https://github.com/user-attachments/assets/5b31e41f-dd1a-4575-900b-9bf2c62da795">

Skia Desktop
<img width="108" alt="OpenSans - Skia Desktop" src="https://github.com/user-attachments/assets/18d1fe4c-31a7-4fff-88d4-04d88b7564f8">

Skia Linux
<img width="83" alt="OpenSans - Skia Linux" src="https://github.com/user-attachments/assets/75d9bd6e-4ebe-4ff5-bf8c-990af3bccf55">

**Comment by ArchieCoder** [comment]
Setting the FontWeight to Normal makes the render much better.

<img width="192" alt="image" src="https://github.com/user-attachments/assets/427c7a6a-81a6-4b92-aaf3-09c32931e654">
